lib/cfndsl/orchestration_template.rb

```diff
@@ -121,7 +123,7 @@ def valid_ref?(ref, origin = nil)
 
       return true if GLOBAL_REFS.key?(ref)
 
-      return true if @Parameters && @Parameters.key?(ref)
+      return true if @Parameters&.key?(ref)
```

Comment:
docs currently state we work with ruby2.1 this syntax would make our minimum ruby2.3